lib/rspec/rails/example/view_example_group.rb

```diff
@@ -150,10 +150,12 @@ def _default_render_options
           match = path_regex.match(_default_file_to_render)
 
           render_options = {template: match[:template]}
-          render_options[:handlers] = [match[:handler]] if match[:handler]
+
+          # remove stringified parts when dropping rails-4.x support
+          render_options[:handlers] = [match[:handler], match[:handler].to_sym] if match[:handler]
```

Comment:
We don't want to match both, we just want to match the right type for each rails version.